_maps/map_files/TgStation/tgstation.2.1.3.dmm

```diff
@@ -22358,6 +22358,12 @@
 	dir = 2;
 	icon_state = "pipe-c"
 	},
+/obj/machinery/button/door{
+	id = "datboidetective";
```

Comment:
it's an internal tag id, only admins will ever see it if they viewvar the door button or door